VideoCapture.py

```diff
@@ -55,15 +58,16 @@
         barcodeType = barcode.type
             # draw the barcode data and barcode type on the image
         text = "{} ({})".format(barcodeData, barcodeType)
-        center = "%d , %d" % (((x+w)/w),((y+h)/h))
+        center = "%d , %d" % ((x+w)/2,(y+h)/2)
         text = text + center
         cv2.putText(frame, text, (x, y - 10),
             cv2.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 255), 2)
             # if the barcode text is currently not in our CSV file, write
             # the timestamp + barcode to disk and update the set
-        if barcodeData not in found:
-            csv.write("{},{}\n".format(datetime.datetime.now(),
-                barcodeData,x,w,y,h))
+        timenow = datetime.datetime.now()
+        if barcodeData == 'Aurora Aerial':
```

Comment:
Can we out this in a global variable so it's easier to access?

Reply:
Yes